src/System.Text.Encodings.Web/tests/EncoderExtensionsTests.cs

```diff
@@ -2,13 +2,14 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.Framework.WebEncoders;
+using NewHtmlEncoder = System.Text.Encodings.Web.HtmlEncoder;
+using NewUrlEncoder = System.Text.Encodings.Web.UrlEncoder;
```

Comment:
I'd very much prefer we not do this; it's inconsistent with how we write tests in general in corefx. And even if we did, the "New" part here doesn't make sense to me. Please revert this.

Reply:
@stephentoub Well, the namespace `System.Text.Encodings.Web.Tests` has file `TemporaryEncoderAdapters.cs` which defines `internal sealed class HtmlEncoder` and `internal sealed class UrlEncoder`, and test is testing classes from `System.Text.Encodings.Web`. Idea for those "aliases" I got from the `PerformanceTests.cs` in the same namespace. What would be the right way of solving this naming problem from the perspective of someone who has expertise like you? Just wondering, I would like to do it the right way and meet `corefx's` standard, that would make me more happy that just reverting, if that is an option :blush:

Reply:
The classes in file TemporaryEncoderAdapters.cs  are temporary and we'll remove it later. we can just have a different namespace in TemporaryEncoderAdapters.cs different than file TemporaryEncoderAdapters.cs 

---
In reply to: [313239124](https://github.com/dotnet/corefx/pull/40238#discussion_r313239124) [](ancestors = 313239124)